src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -197,7 +197,7 @@ function getDownloadObject<T>(columns: Column<T>[], rowData: T) {
 }
 
 class LazyMobXTableStore<T> {
-    @observable public filterString:string;
+    public filterString:string|undefined;
```

Comment:
seems like we were not observing this observable `filterString` anywhere. so, it is now repurposed to restore the `dataStore.filterString` value after component receive new props.